docs/user-level.md

```diff
@@ -197,7 +197,7 @@ BTC RPC Explorer is a very powerful Bitcoin explorer that uses your own full nod
 * Tor onion domain is generated by default.
 * Install `selfhost-clearnet` to activate clearnet domain **continue reading the docs!**
 * Installation of `selfhost-clearnet` **requires interaction** because it asks for domain.
-* **You must have a working domain pointing to your node for `selfhost-clearnet` to work!**
+* **You must have a working domain pointing to your node for `selfhost-clearnet` to work!** Otherwise you can remove `selfhost-clearnet-certbot` to prevent certificate retrieval error during package configuration
```

Comment:
Maybe instead of "Otherwise" write "If you installed it by mistake", I think it should be clearer. What do you think?

Reply:
In my case `selfhost-clearnet-certbot` wasn't installed by mistake. It just comes together with `selfhost-clearnet`. Just checked it in docker:
![image](https://user-images.githubusercontent.com/45279092/144714846-a08894e3-3625-4c07-941a-25f200a39a33.png)

Reply:
Well, `selfhost-clearnet` doesn't work with working domain either (unless you path `/etc/hosts` but that's an edge case), so even installing that one is probably a mistake.

Reply:
Probably it is an undocumented hack, but `selfhost-clearnet` works fine if you provide ipv4 address as the domain name. The only issue is that `selfhost-clearnet-certbot` fails to configure in such case. So, removing `selfhost-clearnet-certbot` makes dpkg happy and keeps `selfhost-clearnet` available by ip address via http. It's definetly not a secure way to use selfhost dashboard, but the esiest to get it up and running. Maybe it makes sense to describe this way in documentation as possible but not recommended?

Reply:
Oh, it didn't occur to me it could be possible. I agree with documenting it is possible but a bad idea due to security issues. I also wouldn't be surprised if some applications didn't work.